pkg/testsuite/snapshotcreator/snapshotcreator.go

```diff
@@ -120,6 +121,10 @@ func CreateSnapshot(opts ...options.Option[Options]) error {
 	)
 	defer engineInstance.Shutdown()
 
+	if opt.AddGenesisRootBlock {
+		engineInstance.EvictionState.AddRootBlock(api.ProtocolParameters().GenesisBlockID(), genesisCommitment.ID())
```

Comment:
If this option is not provided the genesis root block in the eviction state will default to `NewBlockID(p.basicProtocolParameters.GenesisSlot, EmptyIdentifier)` which is the same you are calling here.
```
// GenesisBlockID defines the block ID of the genesis block.
func (p *V3ProtocolParameters) GenesisBlockID() BlockID {
	return NewBlockID(p.basicProtocolParameters.GenesisSlot, EmptyIdentifier)
}
```

Am I missing something?

Reply:
Correct. IMHO the default in the eviction state should be changed. We should not default to a genesis root block at any point in time.